<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>564</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>913</width>
    <height>564</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>913</width>
    <height>564</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>pyLottomanager</string>
  </property>
  <widget class="QWidget" name="Lottozahlen">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>320</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>Lottozahlen</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#Rahmen { border-image: url(:/lottokugel.png);}
QLabel {	background-color: rgb(195, 195, 195)}</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>444</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QPushButton" name="btn_hinzu">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to database</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
   <widget class="QLabel" name="lab_zusatz">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>add. number</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_superz">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="lab_superz">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Super number</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_spiel77">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>384</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="lab_spiel77">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>384</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>game 77</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_super6">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>384</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="lab_super6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>384</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Super 6:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="com_modus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Winning numbers</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>lottery ticket</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_zufall">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>random numbers</string>
    </property>
   </widget>
   <widget class="QComboBox" name="com_laufzeit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>384</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>unlimited</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 draw</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 draws</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 draws</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 draws</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 draws</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lab_laufzeit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Running Time:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_calender_today">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Setting the
date of today</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_scheinnr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>384</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>serial number</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="com_laufzeit_tag">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>384</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>We + Sa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wednesday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Saturday</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>-1</y>
      <width>421</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Winning numbers</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="edi_daten_gewinnz">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="whatsThis">
          <string extracomment="Ausgabefeld">Ausgabefeld</string>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_daten_gewinnz">
         <property name="text">
          <string>selected record</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="Btn_gz_anzeigen">
           <property name="text">
            <string>show</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Btn_gz_loeschen">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="CBox_gz_kompl_ausgeben">
           <property name="text">
            <string>all entries</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>lottery ticket</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="edi_daten_lottoschein">
         <property name="font">
          <font>
           <family>Cantarell</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="whatsThis">
          <string extracomment="Ausgabefeld">Ausgabefeld</string>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_daten_lottoschein">
         <property name="text">
          <string>selected record</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="btn_ls_auswerten">
           <property name="text">
            <string>evaluate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Btn_ls_anzeigen">
           <property name="text">
            <string>show</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Btn_ls_loeschen">
           <property name="text">
            <string>delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>326</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_tag">
       <property name="text">
        <string>day</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinbox_tag">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>31</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_monat">
       <property name="text">
        <string>month</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_monat">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>12</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_jahr">
       <property name="text">
        <string>year</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_jahr">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <number>1900</number>
       </property>
       <property name="maximum">
        <number>2300</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_kalender">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>460</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Open
calendar</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>913</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGo_to_the_website"/>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuHilfe"/>
  </widget>
  <action name="actionBeenden">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionGo_to_the_website">
   <property name="text">
    <string>Go to the website</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>edi_daten_gewinnz</tabstop>
  <tabstop>btn_hinzu</tabstop>
  <tabstop>com_modus</tabstop>
  <tabstop>btn_zufall</tabstop>
  <tabstop>com_laufzeit</tabstop>
  <tabstop>spinBox_superz</tabstop>
  <tabstop>spinBox_spiel77</tabstop>
  <tabstop>spinBox_super6</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
